axelor-account/src/main/java/com/axelor/apps/account/service/batch/BatchRealizeFixedAssetLine.java

```diff
@@ -34,14 +34,20 @@
 import com.axelor.inject.Beans;
 import com.google.inject.Inject;
 import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
+import java.util.Set;
 
 public class BatchRealizeFixedAssetLine extends AbstractBatch {
 
   private FixedAssetLineMoveService fixedAssetLineMoveService;
   private AppBaseService appBaseService;
   private FixedAssetLineService fixedAssetLineService;
+  private final Set<FixedAsset> setFixedAssets = new HashSet<>();
+  private final Map<Integer, Integer> typeCountMap = new HashMap<>();
```

Comment:
not sure about all these private fields, we should make them protected so a custom project can change this batch behavior if needed.